pkg/bindings/connection.go

```diff
@@ -108,23 +111,76 @@ func NewConnectionWithIdentity(ctx context.Context, uri string, identity string,
 	var connection Connection
 	switch _url.Scheme {
 	case "ssh":
+		userinfo := _url.User
+		if _url.User == nil {
+			u, err := user.Current()
+			if err != nil {
+				return nil, fmt.Errorf("current user could not be determined: %w", err)
+			}
+			userinfo = url.User(u.Username)
+		}
 		port := 22
 		if _url.Port() != "" {
 			port, err = strconv.Atoi(_url.Port())
 			if err != nil {
 				return nil, err
 			}
 		}
+		// ssh_config
+		alias := _url.Hostname()
+		logrus.Debugf("Alias: %s", alias)
+		cfg := ssh_config.DefaultUserSettings
+		if val := cfg.Get(alias, "User"); val != "" {
+			userinfo = url.User(val)
+			logrus.Debugf("User: %s", val)
```

Comment:
These debug statements are not clear, they need to mention ssh config at least

Reply:
They could be removed altogether, they were mostly used during implementing...

Reply:
There is likely value in having them to see what the lib parsed in case of weird user configs

Reply:
A compromise might be to log the new URL (once), once all the substitution (if any) is done.

Reply:
Yes logging the final url once sounds good to me.